hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -775,6 +775,29 @@ private Constants() {
       "fs.s3a." + Constants.AWS_SERVICE_IDENTIFIER_STS.toLowerCase()
           + ".signing-algorithm";
 
+  /** Prefix for S3A client-specific properties. */
```

Comment:
can you add an {@value} tag for this and the strings below, for better IDE experience